db/migrate/20150513034751_add_repository_branches.rb

```diff
@@ -0,0 +1,26 @@
+class AddRepositoryBranches < ActiveRecord::Migration
+  def up
+    create_table :katello_repository_branches do |t|
+      t.string :branch, :null => false
```

Comment:
@daviddavis one of the suggestions from @jlsherrill and @ehelms was rename repository_branches table to ostree_repository_branches and do the same for the model.. Also the other suggestion was to call the branch attribute 'ref' . Look for ref in https://github.com/projectatomic/rpm-ostree/blob/master/doc/compose-server.md#TreefileKeyDocumentation